tests/test_generator_declarative.py

```diff
@@ -1509,3 +1501,34 @@ class Simple(Base):
 server_default=text("'test'"))
 """,
     )
+
+
+def test_table_with_arrays(generator: CodeGenerator) -> None:
+    _ = Table(
```

Comment:
Why assign the table if you're not using the variable?
```suggestion
    Table(
```

Reply:
The underscore assignment was just habit - my LSP of choice is basedpyright which I've setup to complain if some function returns a value that I then don't check or use. Thanks for correcting it.